src/main/java/org/jitsi/jicofo/util/JingleOfferFactory.java

```diff
@@ -388,8 +388,10 @@ private void addVideoToContent(ContentPacketExtension content,
         }
 
         // a=rtpmap:107 H264/90000
-        PayloadTypePacketExtension h264
-            = addPayloadTypeExtension(rtpDesc, H264_PT, Constants.H264, 90000);
+        PayloadTypePacketExtension h264 = addPayloadTypeExtension(
+                // XXX(gp): older Chrome versions (users have reported 53/55/61)
+                // fail to enable h264, if the encoding name is in lower case.
+                rtpDesc, H264_PT, Constants.H264.toUpperCase(), 90000);
```

Comment:
why not just change the constant?

Reply:
Yeah, I thought about that but.. what if we break something in the Jitsi client? I'd be a little hesitant to change it there.